hlx_statics/styles/styles.css

```diff
@@ -854,3 +854,23 @@ main .heading6 h6 {
 .warning-icon{
   color: rgb(110, 110, 110);
 }
+
+.spectrum-Divider--sizeM {
+  --spectrum-divider-height: 2px;
+  --spectrum-divider-vertical-height: 2px;
+  --spectrum-divider-vertical-width: 2px;
+}
+
+.spectrum-Divider {
+  --spectrum-divider-vertical-height: 100%;
+  border: none;
+  border-radius: var(--spectrum-divider-height);
```

Comment:
should we flatten these too? (i.e. replace with values to remove dependency on spectrum.min)

Reply:
Okay @melissag-ensemble